CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [3.7.0] - 2019-11-22
+### Added
+- Support **Paste**
+
```

Comment:
Thanks, but undo file changes. Whenever I update the extension, I like to review its content.